packages/tools/devtools/devtools-core/src/data-visualization/VisualTree.ts

```diff
@@ -74,6 +74,11 @@ export interface VisualNodeBase {
 	 * Inside of the porperty is an array of possible EditTypes to inform devtools-view to only show the corresponding edit options for the types allowed
 	 */
 	editProps?: { editTypes?: EditType[] };
+
+	/**
+	 * Schema data of the SharedTree
+	 */
+	sharedTreeSchemaData?: string | Record<string | number, string>;
```

Comment:
We shouldn't have anything dds-specific in the `VisualTree` domain. `VisualTree`s describe how the view should render a generic metadata tree.

To make this more general, we could call it `tooltipContents` or something like that. The contract being that the renderer should display these contents behind a tooltip-like user interaction (like clicking on an icon). And this could contain an entire `VisualTree` if we wanted. Fine to start with just a string for now, though.